tests/testHelpers.ts

```diff
@@ -41,8 +45,10 @@ export function createIndividualApplicationWithSelfieVerification(unit: Unit) {
             fullName: createFullName("Richard", "Hendricks"),
             dateOfBirth: "2001-08-10",
             address: createAddress("20 Ingram St", null, "Forest Hills", "CA", "11375", "US"),
+            nationality: "US",
             email: "april@baxter.com",
             phone: createPhone("1", "5555555555"),
+            occupation: "GigWorker",
```

Comment:
This is a valid occupation. 

https://docs.unit.co/types#occupation

Reply:
Yes that's right, the enum is pulled from the doc. Sorry could you clarify what needs to be changed here exactly?